.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-on: [push]
+on: [push, pull_request]
```

Comment:
😆 is this going to cause CI to fire twice on PRs though, like it was doing earlier?

Reply:
Yeah, which crumby behavior do we want? Feels like going back to a world where renovate PRs don't fire is pretty suboptimal.